src/Compilers/Core/Portable/Operations/IIncrementExpression.cs

```diff
@@ -19,9 +17,29 @@ public interface IIncrementExpression : IOperation, IHasOperatorMethodExpression
         IOperation Target { get; }
 
         /// <summary>
-        /// Kind of increment.
+        /// <code>true</code> if this is a decrement expression.
+        /// <code>false</code> if this is an increment expression.
+        /// </summary>
+        bool IsDecrement { get; }
+
+        /// <summary>
+        /// <code>true</code> if this is a postfix expression.
+        /// <code>false</code> if this is a prefix expression.
+        /// </summary>
+        bool IsPostfix { get; }
+
+        /// <summary>
+        /// <code>true</code> if this is a 'lifted' increment operator.  When there is an 
+        /// operator that is defined to work on a value type, 'lifted' operators are 
+        /// created to work on the <see cref="System.Nullable{T}"/> versions of those
+        /// value types.
+        /// </summary>
+        bool IsLifted { get; }
+
+        /// <summary>
+        /// <code>true</code> if overflow checking is performed for the arithmetic operation.
         /// </summary>
-        UnaryOperationKind IncrementOperationKind { get; }
+        bool IsChecked { get; }
```

Comment:
>ool IsChecked { get; } [](start = 9, length = 22)

Do we have positive and negative tests for this property?

Reply:
Done.